### PR TITLE
Changes to PractitionerRole, Practitioner and Organization examples, addition of Location example

### PR DESCRIPTION
**Pull request by dbojicic**
Related to https://github.com/hl7au/au-fhir-core/issues/132
- Add new 'Location/murrabit-hospital'
- Updated location reference in encounter-discharge
- Added location reference to cardiologist and surgeon practitionerrole examples
- removed unused location examples 
- update ig.xml

### CODE REVIEW

File: input/examples/practitioner-darleen-mackay.xml [hunk 1]
```diff
@@ -4,14 +4,24 @@
...
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="EN"/>
+            </coding>
+            <text value="Employee number"/>
+        </type>
+        <system value="http://nephrologyclinic.org.example.com/employee-id"/>
+        <value value="N765009-mackay-darleen"/>
+        <assigner>
+            <display value="Bobrester Medical Center"/>
```
Comment (dt-r) [inline input/examples/practitioner-darleen-mackay.xml:17]: Wrong assigner. Darleen doesn't work for the Bobrester Medical Center

File: input/examples/practitioner-darleen-mackay.xml [hunk 2]
```diff
@@ -4,14 +4,24 @@
         <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
     </meta>
     <identifier>
-        <system value="http://www.acme.org/practitioners"/>
-        <value value="254567"/>
+        <type>
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="EN"/>
```
Comment (dt-r) [inline input/examples/practitioner-darleen-mackay.xml:10]: Code should be EI

File: input/examples/practitioner-bobrester-bob.xml [hunk 3]
```diff
@@ -7,25 +7,16 @@
     <identifier>
         <type>
             <coding>
-                <system value="http://terminology.hl7.org.au/CodeSystem/v2-0203"/>
-                <code value="EI"/>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="EN"/>
```
Comment (dt-r) [inline input/examples/practitioner-bobrester-bob.xml:11]: Change code back to EI as per [AU Employee Number](https://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-employeenumber.html)

Comment (dbojicic) [reply input/examples/practitioner-bobrester-bob.xml:11]: done

File: input/ig.xml [hunk 4]
```diff
@@ -300,34 +308,34 @@
...
 		</resource>
 		<resource>
-			<reference>
-				<reference value="Location/patient-home"/>
+		<reference>
+				<reference value="Location/mitchells-hill-audiology"/>
 			</reference>
-			<name value="Location - Patient home"/>
-			<description value="Patient home as kind of location (not a specific patient's location)."/>
+			<name value="Location - Murrabit Public Hospital"/>
```
Comment (dt-r) [inline input/ig.xml:337]: Wrong name, this entry is for Mitchells Hill Audiology

File: input/examples/practitioner-darleen-mackay.xml (hunk 1 above)
Comment (dbojicic) [reply input/examples/practitioner-darleen-mackay.xml:17]: 1. she doesn't. changed. 
2. done.

File: input/ig.xml (hunk 4 above)
Comment (dbojicic) [reply input/ig.xml]: Done

File: input/examples/practitioner-darleen-mackay.xml [hunk 5]
```diff
@@ -4,14 +4,24 @@
...
+            <coding>
+                <system value="http://terminology.hl7.org/CodeSystem/v2-0203"/>
+                <code value="EI"/>
+            </coding>
+            <text value="Employee number"/>
+        </type>
+        <system value="http://nephrologyclinic.org.example.com/employee-id"/>
+        <value value="N765009-mackay-darleen"/>
+        <assigner>
+            <display value="Nephrology Clinic"/>
```
Comment (dt-r) [inline input/examples/practitioner-darleen-mackay.xml:17]: reads like type of organisation not name. Make it Docker River Nephrology Clinic.

**Review by dt-r — APPROVED** [review]

**Pull request merged**